webpack.config.js

```diff
@@ -0,0 +1,55 @@
+var webpack = require('webpack');
+var path = require('path');
+var env = process.env.NODE_ENV || 'development';
+
+
+var eslintLoader = {
+  test: /\.js$/,
+  loaders: ['eslint'],
+  include: path.resolve('./source')
+};
+
+
+module.exports = {
+  devtool: 'sourcemap',
+
+  entry: './source/index.js',
+
+  output: {
+    filename: 'index.js',
+    path: path.resolve('./build')
+  },
+
+  plugins: [
+    new webpack.DefinePlugin({
+      'process.env': {
+        NODE_ENV: '"' + env + '"'
+      }
+    })
+  ],
+
+  module: {
+    preLoaders: env === 'development' ? [
+      eslintLoader
+    ] : [],
+    loaders: [
+      {
+        test: /\.js$/,
+        loaders: ['babel?plugins=object-assign'],
+        include: path.resolve('./source')
+      }
+    ]
+  },
+
+  resolve: {
+    extensions: ['', '.js']
```

Comment:
I am personally against separating jsx and js. And keep all React component in normal .js files. 
The reason for that is that I use JSX for modules composition. Not only for UI components. For me any JSX tag with attributes is just a function call with arguments. 
Thinking this way I see no way ot separate them, because any time I can add JSX anywhere in my project.

Besides that we do not have any jsx in here. And as fro #21 this config should stay lean.